tests/database/test_db.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from sqlalchemy import create_engine, text
+from sqlalchemy.orm import sessionmaker, scoped_session
+from harvester.database.models import Base 
+from harvester.database.interface import HarvesterDBInterface
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+@pytest.fixture(scope='session')
+def db_session():
+    DATABASE_SERVER = os.getenv("DATABASE_SERVER")
+    DATABASE_URI = os.getenv("DATABASE_URI")
+    TEST_SCHEMA = "test_schema"
+    modified_uri = DATABASE_URI.replace('@' + DATABASE_SERVER, '@localhost')
+    engine = create_engine(modified_uri)
+
+    with engine.connect() as connection:
+        connection.execute(text(f"CREATE SCHEMA IF NOT EXISTS {TEST_SCHEMA};"))
+        connection.execute(text(f"SET search_path TO {TEST_SCHEMA};"))
+
+    Base.metadata.create_all(engine)
+    SessionLocal = sessionmaker(bind=engine)
+
+    session = scoped_session(SessionLocal)
+    yield session
+
+    session.remove()
+    engine.dispose()
+
+    with engine.begin() as connection:
+        connection.execute(text(f"DROP SCHEMA IF EXISTS {TEST_SCHEMA} CASCADE;"))
+
+@pytest.fixture(scope='session')
+def db_interface(db_session):
+    return HarvesterDBInterface(db_session)
+
+def test_add_harvest_source(db_interface):
+    source_data = {'name': 'Test Source',
```

Comment:
why not use test data here?

Reply:
test data file will be removed. That file is just for initial quick test.